includes/fbproduct.php

```diff
@@ -447,6 +467,58 @@ public function get_fb_description() {
 		return apply_filters( 'facebook_for_woocommerce_fb_product_description', $description, $this->id );
 	}
 
+	public function get_rich_text_description() {
+		$rich_text_description = '';
+
+		if ( $this->fb_description ) {
+			$rich_text_description = $this->fb_description;
+		}
+
+		if ( $this->rich_text_description ) {
+			$rich_text_description = $this->rich_text_description;
+		}
+		
+		
+
+		if ( empty( $rich_text_description ) ) {
+			// Try to get description from post meta if fb description has been set
+			$rich_text_description = get_post_meta(
+				$this->id,
+				self::FB_PRODUCT_DESCRIPTION,
+				true
+			);
+		}
+
+		// Check if the product type is a variation and no rich text description is found yet
```

Comment:
Rather then just saying what the check are, better to say what behaviour are we trying to impose, e.g. "In case of a variable product we want to take rich text description of the variant, and when it is not available fallback on the main (parent) product rich text description"